jaeger-thrift/src/test/java/io/jaegertracing/thrift/internal/senders/UdpSenderTest.java

```diff
@@ -132,7 +138,7 @@ public void testAppend() throws Exception {
   }
 
   @Test
-  public void testFlushSendsSpan() throws Exception {
+  public void testFlushAppendsSpan() throws Exception {
```

Comment:
Feel like original name makes sense, the flush function cause the send of the spans, and server side gets the batch.

Reply:
Since its appending before Flush, I thought the new name covers the intent well.
Do avoid the duplicate code, I extracted the Span creation into a method.